portability-core/src/main/java/org/dataportabilityproject/cloud/google/GooglePersistentKeyValueStore.java

```diff
@@ -55,67 +56,89 @@ public GooglePersistentKeyValueStore(Datastore datastore) {
     this.datastore = datastore;
   }
 
+  /**
+   * Puts {@code jobId} in the database and verifies it doesn't already exist. To update the value
```

Comment:
nit, maybe something like:
 Inserts a new {@link PortabilityJob} keyed by {@code jobId}. Throws SOMEEXCPETION if job already exists. To update an existing {@link PortabilitJob}, use {@link #atomicUpdate(...)} method